glom/test/test_path_and_t.py

```diff
@@ -168,6 +168,22 @@ def test_path_items():
     assert Path().items() == ()
 
 
+def test_path_star():
+    val = {'a': [1, 2, 3]}
+    assert glom(val, 'a.*') == [1, 2, 3]
+    val['a'] = [{'b': v} for v in val['a']]
+    assert glom(val, 'a.*.b') == [1, 2, 3]
+    assert glom(val, Path(T['a'], '*', T['b'])) == [1, 2, 3]
+    # multi-paths eat errors
+    assert glom(val, Path('a', '*', T.b)) == []
+    val = [[[1]]]
+    assert glom(val, '**') == [ [[1]], [1], 1]
+    val = {'a': [{'b': [{'c': 1}, {'c': 2}, {'c': 3}]}]}
+    assert glom(val, '**.c') == [1, 2, 3]
+    assert glom(val, 'a.**.c') == [1, 2, 3]
+    assert glom(val, 'a.*.b.*.c') == [1, 2, 3]
+
+
```

Comment:
also need to add some exception tests, what does the PAE look like?

Reply:
oh yea, in general it returns `[]` if there's an error

Reply:
but I guess I could force it with `def __getitem__(): 1/0` kind of thing